src/mapclient/view/workflow/workflowwidget.py

```diff
@@ -210,10 +208,12 @@ def new(self, pmr=False):
 
     def _workflow_finished(self, successfully):
         if successfully:
+            close_after = self._main_window.model().optionsManager().getOption(CLOSE_AFTER) * 1000
```

Comment:
Let's put the scaling from seconds to milliseconds in the self closing message box class.  Then in our "documentation" we say the close after units are in seconds.